service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsAboutToSubmitControllerTest.java

```diff
@@ -178,12 +196,15 @@ void shouldCreateAdditionalApplicationsBundleWithOtherApplicationsBundleWhenOthe
 
         if (servingOthersToggledOn) {
             assertThat(additionalApplicationsBundle.getOtherApplicationsBundle().getOthersNotified())
-                .isEqualTo("Alex Smith");
+                .isEqualTo("Margaret Jones, Alex Smith");
             assertThat(additionalApplicationsBundle.getOtherApplicationsBundle().getOthers())
                 .isEqualTo(List.of(caseData.getOthers().getAdditionalOthers().get(0)));
+            assertThat(additionalApplicationsBundle.getOtherApplicationsBundle().getRespondents())
+                .isEqualTo(List.of(caseData.getRespondents1().get(0)));
```

Comment:
I'd personally prefer to have `assertThat(...).hasSize(1).contains(caseData.getRespondents1().get(0))`

`caseData.getRespondents1().get(0)` could even be constant used when setting the case data as well as here (or a local variable).